spec/spec_helper.rb

```diff
@@ -0,0 +1,19 @@
+require 'coveralls'
+Coveralls.wear!
+
+
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path("../dummy/config/environment.rb",  __FILE__)
+# require 'rspec/rails'
+require 'rspec/autorun'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.